content/en/docs/contribute/generate-ref-docs/kubectl.md

```diff
@@ -235,7 +235,7 @@ Build the Kubernetes documentation in your local `<web-base>`.
 
 ```shell
 cd <web-base>
-make docker-serve
+make container-serve
```

Comment:
Can we probably leave the notion that the docker-serve is still available but deprecated? I know that the docker already mentioned that as well. But writing it on the docs might make it easier to remember when we need to remove the deprecated method in the Makefile